.gitattributes

```diff
@@ -1,4 +1,73 @@
-data/templates/**/*.conf linguist-language=Jinja
-*.tmpl linguist-language=Jinja
-*.xml.i linguist-language=XML
-*.xml.in linguist-language=XML
+# Common settings that generally should always be used with your language specific settings
+
+# Auto detect text files and perform LF normalization
+*          text=auto
+
+#
+# The above will handle all files NOT found below
+#
+
+# Documents
+*.md       text diff=markdown
+*.mdx      text diff=markdown
+*.adoc     text
+*.csv      text eol=crlf
+*.rst      text
+*.txt      text
+
+# Graphics
+*.png      binary
+*.jpg      binary
+*.jpeg     binary
+*.gif      binary
+*.tif      binary
+*.tiff     binary
+*.ico      binary
+# SVG treated as text by default.
+*.svg      text
+# If you want to treat it as binary,
+# use the following line instead.
+# *.svg    binary
+
+# Scripts
+*.bash     text eol=lf diff=bash
+*.sh       text eol=lf diff=bash
+
+# Serialisation
+*.json     text
+*.toml     text
+*.tmpl     text linguist-language=Jinja
```

Comment:
.j2 is missing for Jinja2

Reply:
Right you are, I missed `.j2` and some other extensions. Have added them and tweaked some other ones. Generally, line endings for most text files should be handled properly by git but good to have them explicitly listed.